### DISCUSSION TITLE
Quasar  2.x, q-table data

### DISCUSSION THREAD

**Discussion by JTrotta** [quasarframework/quasar#14217] [General - Components / Directives / etc] [answered]
I'm moving from Quasar 1.x to Quasar 2.x.
Now Quasar 2.x ceates a proxy during data assign. i.e.
`        data: function () {
            return {
                filter: '',
                loading: false,
                separator: 'horizontal',
                pagination: {
                    page: 1,
                    rowsPerPage: 1,
                },
                rowData: []
            };
        },` 
And 

` mounted(){
            let serverRows = @Html.Raw(Newtonsoft.Json.JsonConvert.SerializeObject(Model.Allowed));
            let userRoles = @Html.Raw(Newtonsoft.Json.JsonConvert.SerializeObject(Model.AvailableUserRoles));
            let rows = [];
            let i = 0;
            for (let key in serverRows) {
                item = {}
                let value = serverRows[key];
                item["Id"] = i;
                item["Name"] = key;
                var j = 0;
                for (let cb in value) {
                    if (value[cb] == true)
                        item[userRoles[j].SystemName] = key;
                    else
                        item[userRoles[j].SystemName] = false;
                    j++;
                }
                rows.push(item);
                i++;
            }
            this.rowData = rows;
            this.pagination.rowsPerPage = this.rowData.length;
        }`
this.rowData is a proxy no more an array. 
How to assign the proxy to q-table :data ?

**Comment by metalsadman** [comment, marked as answer]
`:rows="rowData"`
